src/test/java/com/google/cloud/teleport/templates/common/BigQueryConvertersTest.java

```diff
@@ -157,6 +171,41 @@
   private String dateTimeFieldDesc = "Full publication date";
   private String dateTimeFieldValue = "2013-08-19 23:28:20.000567";
 
+  /** Tests that {@link BigQueryConverters.AvroToMutation} creates a Mutation. */
+  @Test
+  public void testAvroToMutation() {
+    // Create test data
```

Comment:
Could you replace these comments with `// Arrange`, `// Act`, and `// Assert`? That seems to be what the test comments are mapping to, but the Arrange/Act/Assert convention is a bit more consistent with other tests.